pipeline/analyze_assembly.py

```diff
@@ -118,24 +126,34 @@ def load_overlaps(self, filename,  min_ident,  comparison =best_hit, min_length=
 		     self.overlaps_identity[name2] = ident
        		     self.contigs_overlaps[name2] +=1  
 
-        for name2 in self.overlaps_s1  :
-           name1 = self.overlaps_genome[name2]
+        # iterate over all the overlaps,
+        for name2 in self.overlaps_s1:
+
+            # get the best (highest identity) overlap
+            name1 = self.overlaps_genome[name2]
             
-           if (name1 in covered) and (name2 in aligned) and (name1 != "null") and comparison( self.contigs_overlaps[name2],1) :
+            # @CTB when would name1/name2 not be in covered/aligned?
+            # @CTB note, I don't understand how comparison is used here
+            if (name1 in covered) and (name2 in aligned) and (name1 != "null") and comparison( self.contigs_overlaps[name2],1) :
                  cov = covered[name1]
                  align = aligned[name2]
                  s1 = self.overlaps_s1[name2] 
                  e1 = self.overlaps_e1[name2] 
                  s2 = self.overlaps_s2[name2]
                  e2 = self.overlaps_e2[name2] 
                  ident = self.overlaps_identity[name2]
+
+                 # ok, add coverage/# of alignments if alignment passes.
                  if e1 - s1 + 1 >= min_length and ident >= min_ident:
-                    for i in range(s1 -1, e1): 
+                    for i in range(s1 - 1, e1): 
 			cov[i] += 1
-                    for j in range(s2-1, e2):
+                    for j in range(s2 - 1, e2):
 			align[j] +=1
 
-    def load_coords(self, filename,  min_ident, min_length=100):
+    # @CTB ok, how does this differ from previous function?
+    # @CTB I see, this is used for ambiguous, load coords used for
```

Comment:
load-coordinate for ambiguous , load_overlap used for besthit and no-overlap (no misassemblies)